Tasks/VsTest/vstest.ts

```diff
@@ -699,6 +699,11 @@ function invokeVSTest(testResultsDirectory: string): Q.Promise<number> {
                 .then(function (vsVersion) {
                     try {
                         vstestLocation = getVSTestLocation(vsVersion);
+                        vsTestVersionForTIA = getVsTestVersion();
```

Comment:
we assign tiaEnabled to false only if version is less than 15.0.25807 and leave it unchanged otherwise. All calls to isTiaAllowed() happen after this part of the code